apps/common/models/file-upload.js

```diff
@@ -3,10 +3,10 @@
 
 const url = require('url');
 
-const Model = require('hof').model;
+const AuthToken = require('./auth-token');
 const config = require('../../../config');
 
-module.exports = class UploadModel extends Model {
+module.exports = class UploadModel extends AuthToken {
```

Comment:
Little did I know that the HOF Model called which this extends off, calls the auth() function below for POSTing the uploads in the first place even though it isn't explicit in the service code. Eventually I worked that out. In this case we need to share the auth logic with this upload component but I've separated it out so other processes can extend off the AuthToken model but where it's functionality is still accessible here for the HOF model process.